<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Barkod Okuma Uygulamasi</string>
  </property>
  <widget class="QComboBox" name="PortComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 255);
font: 75 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>EKLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeletePushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);
font: 75 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>SÄ°L</string>
   </property>
  </widget>
  <widget class="QLabel" name="SqlSelectLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>BAGLANTI YOK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>681</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdatePushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);
font: 75 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>GUNCELLE</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
